specification.md

```diff
@@ -1055,10 +1057,168 @@ For example:
 }
 ```
 
-Note that the referenced function definition type in this case must be `rest` (default type).
+Note that the referenced function definition type in this case must be `openapi` (default type).
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for RESTful Service Invocations
+
+The specification also supports describing REST invocations in the [functions definition](#Function-Definition) using [OpenAPI Paths Object](https://spec.openapis.org/oas/v3.1.0#paths-object).
+
+Here is an example function definition for REST requests with method `GET` and request target corresponding with [URI Template](https://www.rfc-editor.org/rfc/rfc6570.html) `/users/{id}`:
+
+```json
```

Comment:
is it not possible for operation here to also be a string? link to file that contains the object? also i am not sure if "operation" should be top-level object here but need to think about this more.

Reply:
No, it's not possible to use string, since it's a complex object. We would have to "invent" a new way of expression this complex object in a string. The schema is already forcing it. This is what we agreed upon to have the OpenAPI spec path incorporated.

Reply:
@shaisachs The reason behind this feature is to have the ability to define a rest operation without relying on an OpenAPI spec file. Thus, we incorporated part of the spec in the function definition since it's a well-known spec for REST operations definitions.